install_costar.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+# source: https://gist.github.com/phatblat/1713458
+# Save script's current directory
+DIR=$(pwd)
+
+set -e
+set -u
+set -x
+
+echo "############################"
+echo "# costar_stack ROS library"
+echo "############################"
+echo "# Robot User Interface"
+echo "# for flexible task creation "
+echo ""
+echo "# github.com/cpaxton/costar_stack"
+
+
+# ROS must be installed first, assuming it is in default /opt/ros location
+if [ ! -d /opt/ros ]; then
+    ./ros.sh
+fi
+
+cd ~/src
+
+location="cpaxton" # github.com/ahundt/Tasks # I have some patches here
+#location="lcsr" # github.com/jrl-umi3218/Tasks # ongoing development happens here
+#location="jorisv" # github.com/jorisv/Tasks # original repository location
+
+# TODO(ahundt) switch back to master once standardized cmake changes are merged, see https://github.com/jrl-umi3218/jrl-cmakemodules/pull/103
+branch="master"
+#branch="package" # this branch adds standard cmake package configuration
+
+
+. /etc/lsb-release # get ubuntu version number
+
+
+if [ "$DISTRIB_RELEASE" = "16.04" ]; then
+    ROSVERSION="kinetic"
+	# TODO(ahundt) How to install fcl? should "soem" be installed?
+	# TODO(ahundt) Are there univeral robot ros-industrial kinetic binaries?
+	sudo apt-get install -y ros-kinetic-moveit # ros-kinetic-universal-robot ros-kinetic-ur-msgs #  ros-indigo-fcl
+
+	source /opt/ros/kinetic/setup.bash
+fi
+
+
+if [ "$DISTRIB_RELEASE" = "14.04" ]; then
+    ROSVERSION="indigo"
+	sudo apt-get install -y ros-indigo-moveit-full ros-indigo-fcl ros-indigo-soem
+
+	source /opt/ros/indigo/setup.bash
+fi
+
+# openni2 and friends is optional
+sudo apt install -y libopenni2-0 libopenni2-dev openni2-doc openni2-utils ros-${ROSVERSION}-openni2-camera ros-${ROSVERSION}-openni2-launch
+
+# instructor python dependencies
+sudo apt install -y qt4-designer qt4-dev-tools python-qt4 python-qt4-dev python-wxversion wx-common python-wxgtk3.0
+
+# many of these are required, ${ROSVERSION} will be indigo, kinetic as appropriate
+sudo apt install -y python-catkin-tools liburdfdom-headers-dev ros-${ROSVERSION}-control-msgs ros-${ROSVERSION}-gazebo-ros-control ros-${ROSVERSION}-python-orocos-kdl xdot libccd-dev ros-${ROSVERSION}-ros-control ros-${ROSVERSION}-octomap-msgs ros-${ROSVERSION}-gazebo-plugins ros-${ROSVERSION}-pcl-ros ros-${ROSVERSION}-socketcan-interface ros-${ROSVERSION}-rqt-gui ros-${ROSVERSION}-object-recognition-msgs ros-${ROSVERSION}-realtime-tools ros-${ROSVERSION}-position-controllers ros-${ROSVERSION}-robot-state-publisher ros-${ROSVERSION}-joint-state-controller
```

Comment:
remove apt-get list; rosdep means less special handling for when packages change names